.gitignore

```diff
@@ -1,12 +1,8 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
-composer.lock
-vendor
-node_modules
-.sass-cache
+/.sass-cache
+/build
+/Resources/doc/_build
+/node_modules
```

Comment:
Not sure how and where we update our assets. There are not docs or build script atm :(

I think we should remove these files from the `.gitignore` file for now and create a new PR which adds the missing parts for the assets and the coresponding `.gitignore` entries.

Can you do this @shulard ?